lib/ansible/module_utils/dnos10.py

```diff
@@ -0,0 +1,122 @@
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Files in module_utils (unlike any other files) should be licensed BSD instead of GPL. Some networking modules have slipped through this, but we're trying to make sure it all gets in properly now. See [eos.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/eos.py) for a suitable license header to copy, if you need an example.